src/modules/QtQml/Qt.js

```diff
@@ -105,6 +105,34 @@ const Qt = {
     return new QmlWeb.QSizeF(width, height);
   },
 
+  include(path)
+  {
+    const engine = QmlWeb.engine;
+
+    const uri = engine.$resolvePath(path);
+
+    /* Handle recursive includes */
+    if (QmlWeb.executionContext.$qmlJsIncludes === undefined) {
+      QmlWeb.executionContext.$qmlJsIncludes = [];
+    }
+
+    if (QmlWeb.executionContext.$qmlJsIncludes.indexOf(uri) >= 0)
```

Comment:
Nit: `if (QmlWeb.executionContext.$qmlJsIncludes.indexOf(uri) >= 0) {` (that should fit). Not sure why that isn't linted already.